packages/ui/src/components/vuestic-components/va-button-toggle/VaButtonToggle.vue

```diff
@@ -5,10 +5,11 @@
         v-for="option in options"
```

Comment:
Paddings do not seem correct:
![chrome_2020-10-08_22-38-25](https://user-images.githubusercontent.com/5394573/95505521-21765380-09b7-11eb-9901-7ce0e9c2f7e8.png)

https://app.zeplin.io/project/5bcd9657767a220982a4ab61/screen/5d94b7c712b71a16805b7353